src/core-sdk-tasks/core-sdk-tasks.csproj

```diff
@@ -10,12 +10,12 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Build" Version="15.7.179" />
-    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="15.7.179" />
+    <PackageReference Include="Microsoft.Build" Version="$(MicrosoftBuildPackageVersion)" />
```

Comment:
Why do we need to update these? What prompted this PR?

Reply:
In 6/7, the nuget version was pinned to 5.8 (and flagged by CG) and I noticed that msbuild was also pinned to an older version. So this was me just going through every branch to update the msbuild one in addition to nuget 
(which was already updated in 8)